test/prepare_disk.sh

```diff
@@ -0,0 +1,431 @@
+#!/bin/sh
```

Comment:
you reccon it a good idea to name the script scripts/30_prepare_disk here to improve searchability etc?

Reply:
No i don't think so, i think adding the number for sorting should only be added if this is intended and not for the reason of script names having the exactly tje same filename.